packages/react-geo/rollup.config.js

```diff
@@ -0,0 +1,59 @@
+import { defineConfig } from 'rollup';
+import typescript from '@rollup/plugin-typescript';
+import { terser } from 'rollup-plugin-terser';
+import styles from 'rollup-plugin-styles';
+import externals from 'rollup-plugin-node-externals';
+
+// common config settings
+const input = ['src/index.ts'];
+const sourceMap = false;
+const tsconfig = 'tsconfig.dist.json';
+
+const config = defineConfig([
+  // CJS config
```

Comment:
doesn't need to be addressed here, but I wonder if we should have shared rollup configs.